wordpress-mobile/WordPress-Android
# Lint: Upgrade Lint Baseline

This PR upgrades the lint baseline to latest `5` format, `4.0.2` version. This is done in order to make lint more stable and up-to-date.

To test:
1. Pull the branch locally.
2. Run ` ./gradlew lintWordPressVanillaRelease` and verify everything works as expected.
3. Check CI and verify everything works as expected.

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I run `Lint` in both, `debug` and `release` build types, as well as both, `wordPress` and `jetpack` dimensions to make sure that this change is not affecting the lint report and the actual reported issues, both `errors` and `warnings`.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
